circle-area-task.js

```diff
@@ -0,0 +1,8 @@
+// 4. Napisz funkcję circleArea(diameter), która obliczy pole koła dla podanej średnicy. Użyj potęgowania.
+
+var diameter = 3;
+
+function circleArea(diameter) {
+    return Math.PI * diameter * diameter;
+}
+console.log(circleArea(diameter));
```

Comment:
You don't have to create variables for such simple checks.

Just check this function against for example 3 different numbers.